src/fragments/lib/interactions/js/getting-started.mdx

```diff
@@ -42,69 +42,98 @@ Upon successful execution of the push command, a configuration file called `aws-
 > If your Interactions resources were created with Amplify CLI version 1.6.4 and below, you will need to manually update your project to avoid Node.js runtime issues with AWS Lambda. [Read more](/cli/function/configure-options)
 
 ## Manual setup
-### Lex V1 bot
-You can create Amazon Lex V1 chatbot in Amazon Lex console. To create your bot, follow the steps shown in [Amazon Lex V1 Developer Guide](https://docs.aws.amazon.com/lex/latest/dg/getting-started.html).
 
-![Interactions](/images/interactions_lex_console_edit_bot.jpg)
+### Lex V2 bot
```

Comment:
I think this page was initially designed for lexV1 (amplify cli mentioned above).
When we added lexV2 support we added a callout on top and the lexV2 section.

Since we are default to lexV2 should we slightly restructure and move the cli information inside the lexV1 section

Reply:
sounds good, I will add the interaction CLI setup guide in V1 section.